extensions/empty-screenshots/package.json

```
@@ -50,12 +75,21 @@
               "value": "90"
             }
           ]
+        },
+        {
+          "name": "prefixString",
+          "type": "textfield",
+          "title": "Custom prefix (optional)",
+          "required": false,
+          "description": "You can optionally add a custom prefix for the screenshot filenames.",
+          "placeholder": "Screenshot, CleanShot, etc."
```

🍹 What do you think about this renaming?
```suggestion
          "name": "filePrefix",
          "type": "textfield",
          "title": "File prefix (optional)",
          "required": false,
          "description": "You can optionally add a custom prefix for the screenshot filenames.",
          "placeholder": "Screenshot, CleanShot, etc."
```